src/pybind/mgr/dashboard/config_options.html

```diff
@@ -0,0 +1,114 @@
+
+{% extends "base.html" %}
+
+{% block content %}
+
+
+<script>
+  $(document).ready(function(){
+             // Pre-populated initial data at page load
+            var content_data = {{ content_data }};
+	    
+	     rivets.formatters.display_arrays = function(arr) {
+	         result = arr.join().replace(/,/g, "<br/>");
+	         return result;
+             };
+   
+             apply_filters = function() {
+                 content_data.options_list = [];
+                 var selection = "#" + content_data.service;
+                 $(selection).attr('selected','selected');
+                 var level = $("#level").val();
+                 var service = $("#service").val();
+                 if (level == "any" && service == "any") {
+                     content_data.options_list = content_data.options.options;
+                 }
+                 for (var opt of content_data.options.options) {
+                     if (service == "any" && opt.level == level) {
+                         content_data.options_list.push(opt); 
+                     } else if (opt.services.includes(service) && level == "any") {
+                         content_data.options_list.push(opt);
+                     } else if (opt.services.includes(service) && opt.level == level) {
+                         content_data.options_list.push(opt);
+                     }
+                 }
+         
+             };
+             
+             rivets.bind($("#content"), content_data);         
+             apply_filters();
+
+         });
+</script>
+
+<!-- Page Header -->
+<section class="content-header">
+    <h1 style="font-weight:bold">
+      Configuration Options
+      <div class="pull-right" style="font-size:17px">
+	<label>Services:</label>
+	<select id="service" style="color:grey" onchange="apply_filters()">
+	  <option id="mon">mon</option>
+	  <option id="mgr">mgr</option>
+	  <option id="osd">osd</option>
+	  <option id="mds">mds</option>
+	  <option id="common">common</option>
+	  <option id="mds_client">mds_client</option>
+	  <option id="rgw">rgw</option>
+	  <option id="any">any</option>
+	</select>
+	<label>Level:</label>
+	<select id="level" style="color:grey" onchange="apply_filters()">
+	  <option>basic</option>
+	  <option>advanced</option>
+	  <option>developer</option>
+	  <option>any</option>
+	</select>
+      </div>
+    </h1>
+</section>
+	
+<!-- Main content -->
+<section class="content">
+    <div class="box" style="overflow:auto">
+      <div class="box-body">
+	    <table class="table table-bordered">
+	        <thead>
+		    <tr>
+		        <th>Name</th>
+			<th>desc</th>
```

Comment:
Column headers should be full words where reasonable (i.e. Description rather than dec)